### PR TITLE
fix(#229): add semi-transparent background on 404 message

### PR DESCRIPTION
**Pull request by geekananya**
### Description

I made the following changes to prevent 404 notfound page message from being invisible over some backgrounds.

### Changes Made

- Wrapped the text in another div.
- Added Semi-transparent white background
- Added padding

### CODE REVIEW

File: src/pages/NotFound.jsx [hunk 1]
```diff
@@ -11,22 +11,30 @@ const NotFound = () => {
...
             }}
         >
-            <h1
+            <div
                 style={{
-                    fontSize: '36px',
-                    marginBottom: '20px',
+                    display: 'inline-block',
+                    padding: '20px',
+                    backgroundColor: 'rgb(256, 256,256, 0.3)',
```
Comment (CREDO23) [inline src/pages/NotFound.jsx]: Can you increase once more the opacity, try with 0.45

Comment (CREDO23) [inline src/pages/NotFound.jsx]: Please use tailwind classes here @geekananya

File: src/pages/NotFound.jsx [hunk 2]
```diff
@@ -11,22 +11,30 @@ const NotFound = () => {
...
+                        marginBottom: '20px',
+                    }}
+                >
+                    404 - Page non trouvée
+                </h1>
+                <p
+                    style={{
+                        fontSize: '18px',
+                    }}
+                >
```
Comment (CREDO23) [inline src/pages/NotFound.jsx]: Use tailwind CSS

File: src/pages/NotFound.jsx [hunk 3]
```diff
@@ -11,22 +11,30 @@ const NotFound = () => {
...
+                    {"Désolé, la page que vous recherchez n'existe pas."}
+                </p>
+                <Button text={'Go to home'} event={handleClick} />
```
Comment (CREDO23) [inline src/pages/NotFound.jsx]: This button is invisible in the page ! Please, make it visible ...

**Review by CREDO23 — CHANGES_REQUESTED** [review]
@geekananya, It looks good ! Thanks.
Just add the requested changes !

Also, try to apply the opacity to the full page and center text content, or keep the opaque box as is and center it in the screen !

**Comment by geekananya** [comment]
Hi, @CREDO23, thanks for the review! I will apply the new changes and get back to you.

**Comment by geekananya** [comment]
@CREDO23, I made the changes as requested, please review the updates and let me know if any modifications are needed.

**Review by CREDO23 — APPROVED** [review]
Thanks for your contribution @geekananya !

**Pull request merged**